test/test_helper.rb

```diff
@@ -68,6 +72,54 @@ def stub_module(name)
   end
 end
 
+# helper for defining missing YaST modules,
+# these needs to be done *after* calling SimpleCov.start so the coverage
+# of the imported modules is correctly counted
+def define_if_missing(name, &block)
+  # try loading the module, it might be present in the system (running locally
+  # or in GitHub Actions), mock it only when missing (e.g. in OBS build)
+  Yast.import name
+  puts "Found module Yast::#{name}"
+rescue NameError
+  warn "Mocking the Yast::#{name} module completely"
+  block.call
+end
```

Comment:
maybe we can add it to yast/rspec in ruby bindings? I think we face it in more modules and we handle it quite inconsistently.